src/main/java/duke/DukeException.java

```diff
@@ -0,0 +1,14 @@
+package duke;
+
+/**
+ *  A class which encapsulates the exceptions that Duke throws
+ *  @author  Chen Guanzhou
+ *  @version v1
+ */
+
```

Comment:
Perhaps no blank line between the documentation block and the class?

Reply:
Thank you